.env.test

```diff
@@ -0,0 +1,3 @@
+SHOW_HEADER=false
+HEADER_ASCII_ART=false
+SIMPLE_OUTPUT=false
```

Comment:
If I remember correctly, I think we commented that this .env should be in the acceptance test folder.